lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3716,9 +3716,21 @@ namespace Js
         // Debugger attach/detach failure is catastrophic, take down the process
         DEBUGGER_ATTACHDETACH_FATAL_ERROR_IF_FAILED(hr);
 
+        HRESULT hrEntryPointUpdate = S_OK;
+        BEGIN_TRANSLATE_OOM_TO_HRESULT_NESTED
```

Comment:
nit: other usages of this macro pair wrap the body with an open brace and close brace to make the scope more obvious